tangelo/linq/qpu_connection/ibm_connection.py

```diff
@@ -0,0 +1,140 @@
+"""
+    Wrappers around Qiskit runtime API, to manage quantum experiments run with IBM Cloud or IBM quantum
+    from Tangelo.
+"""
+
+import os
+
+from tangelo.linq.translator import translate_qiskit
+from tangelo.linq.qpu_connection.qpu_connection import QpuConnection
+
+try:
+    from qiskit.providers.jobstatus import JobStatus
+    from qiskit_ibm_runtime import QiskitRuntimeService
+    is_qiskit_installed = True
+except ModuleNotFoundError:
+    is_qiskit_installed = False
```

Comment:
Nice! I think I will take a similar approach for the `installation without pyscf` task. We can also change the encapsulation of `qemist_client` in the future!